src/components/app/game-board/GameBoard.tsx

```diff
@@ -17,6 +18,13 @@ function GameBoard({ className }: ChildProps): React.JSX.Element{
         totalGameTime: 0
     });
 
+    const setOpener: (game: Game) => string = useCallback((game:Game) => {
```

Comment:
Looks like it should be a 'getOpener'. And You can remove the type for what the function returns :)